lib/core/AxiosHeaders.js

```diff
@@ -222,13 +216,13 @@ Object.assign(AxiosHeaders.prototype, {
     return this;
   },
 
-  toJSON: function() {
+  toJSON: function(asStrings) {
```

Comment:
When is toJSON used? Cannot see anywhere where it is used with `asString === true`?
Are there cases where one wants arrays to be returned as `string`?

Reply:
> When is toJSON used?

at least in test cases to make deep assertions, but...
> asString === true

AxiosHeaders is a public class, so not 100% of its functionality should be used inside the Axios codebase, from the user's point of view, it should be a complete helper implementation for dealing with headers.